.github/styles/Vocab/Docs/accept.txt

```diff
@@ -115,4 +115,6 @@ wget
 Workbench
 Zabbix
 Zipkin
-ZooKeeper
\ No newline at end of file
+ZooKeeper
+redli
+[Ss]tunnel
```

Comment:
The same comment here. The product is "Stunnel", and you can also use the command `stunnel` if you need to - but with literal/code markup and not as a word, so the spellchecker should not recognise the all-lowercase version of the word.